design/20190708.certificate-request-crd.md

```diff
@@ -427,50 +433,21 @@ this resource.
 
 #### Issuing Controller
 
-Since external issuers have been built before the addition of Approved and
-Denied conditions, the issuing controller needs to be permissive. An external
-issuer may not honour an Approved condition and will sign and set a
-CertificateRequest as being Ready, before the request has been approved. The
-issuing controller must mark issuance as being successful in this case. In
-practice, this means that the issuing controller is never concerned with
-Approved conditions.
-
-External issuers that do not honour Denied conditions will sign
-CertificateRequests, even if they have a Denied condition set. In this case, the
-issuing controller will successfully complete the issuance of the Certificate.
-
-
-External Issuers and internal issuers that honour the Denied condition will
-never sign CertificateRequests with the Denied condition set, and thus never set
-Ready condition. In this case, the issuing controller will consider this
-CertificateRequest as failed, and will set the condition `Issuing=False`
-as well as setting the status field `lastFailureTime`. Note that the issuing
-controller is not responsible for setting the `Ready=False` condition on
-the CertificateRequests; that's the issuer's responsibility.
-
-- The Certificate is clearly reported as Failed to users who may miss the
-    Denied request from a cursory view.
-- The Spec may genuinely be violating the policy, and so can be changed by the
-    user. This will cause an immediate reissue.
-- The policy may be misconfigured, and as such, the Certificate will be retired
-    later with no user intervention.
-- The [manual renew
-    command](https://cert-manager.io/docs/usage/kubectl-plugin/#renew) relies on
-    the Issuing condition. In the case of policy being misconfigured, the user
-    is able to immediately retry the request using the CLI plugin.
+Issuing controller considers all Denied CertificateRequests to be in a final failed state.
+The issuance will be failed and will be continuously retried with an exponential backoff ../20220118.certificate-issuance-exponential-backoff.md.
```

Comment:
```suggestion
The issuance will be failed and will be continuously retried with an [exponential backoff](../20220118.certificate-issuance-exponential-backoff.md).
```

I'm not a fan of `continuously`. exponential backoff isn't continuous, it's exponential. Possibly `repeatedly` or `until it reaches a terminal state`...